dotnetOQS/KEM.cs

```diff
@@ -77,12 +80,9 @@ static KEM()
                 string alg = Marshal.PtrToStringAnsi(OQS_KEM_alg_identifier(i));
                 supported.Add(alg);
                 // determine if the alg is enabled
-
-                IntPtr kemPtr = OQS_KEM_new(alg);
-                if (kemPtr != IntPtr.Zero)
+                if (OQS_KEM_alg_is_enabled(alg) == 1)

```

Comment:
@christianpaquin I'd just replace this with `if (OQS_KEM_alg_is_enabled(alg))`, so we don't rely on a hardcoded `1` (just in case we may later decide to return some other non-zero status code from `OQS_KEM_alg_is_enabled`.

Reply:
Right, that would be better. I initially was checking for OQS_SUCCESS, but my tests were failing (I was surprised we didn't use that returned value here).

Reply:
@christianpaquin I remember I raised this question, whether we shall have a status code such as OQS_ENABLED or so. Right now we consider that as a pure boolean function (which for now is OK), but in the future we may need to change the return status. So I'd say let's go with this as is right now (i.e. return 0 or 1), and maybe figure out later what we want to do about it.

Reply:
Yes, if we change the OQS API to return enable/supported, then we should use this here as well. As far as removing " == 1", I forgot that C# prevents implicit conversion between int and bool, so I can't remove the check.

Reply:
@christianpaquin What about then doing `!= 0` instead?

Reply:
@vsoftco, would that be very different? I like the explicit nature of checking 1. If we change the OQS API we'll have to update this project anyway.

Reply:
@christianpaquin Oh I see. Agreed, let's leave it as is then.